test-cases/gutenberg/group.md

```diff
@@ -52,12 +52,12 @@ Expected look:
 
 Expected look:  
 ![PlaceholderNestes](../resources/group-placeholder-nest-1.png)
- - Post title is not dimmed
+ - Post title **is not** dimmed
```

Comment:
We can remove these lines, we can accept the not dimming as the default behavior anymore.

Reply:
line removed